subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/DefaultArtifactCaches.java

```diff
@@ -59,6 +59,7 @@ public DefaultArtifactCaches(CacheScopeMapping cacheScopeMapping,
             if (baseDir != null) {
                 readOnlyCacheMetadata = new DefaultArtifactCacheMetadata(cacheScopeMapping, baseDir);
                 readOnlyArtifactCacheLockingManager = new ReadOnlyArtifactCacheLockingManager(cacheRepository, readOnlyCacheMetadata);
+                LOGGER.info("The read-only dependency cache is enabled \nThe "+ READONLY_CACHE_ENV_VAR + " environment variable was set to " + baseDir);
```

Comment:
It is probably not a big deal in this particular case, but in general, it is better to use `.info(String format, Object ...args)` methods so that String concatenation will only happen inside the logger if the particular logging level is enabled.

Reply:
addressed this, thanks for the suggestion